extra_data/cli/make_virtual_cxi.py

```diff
@@ -55,8 +55,8 @@ def main(argv=None):
     ap.add_argument(
         '--fill-value', action='append', nargs=2, metavar=('DS', 'V'),
         help='define fill value (V) for individual dataset (DS). Datasets are'
-             ' "data", "gain" and "mask". (defaults: data: nan; gain: 0; '
-             ' mask: 0xffffffff)'
+             ' "data", "gain" and "mask". (defaults: data: nan (float32) or 0'
+             ' (uint16); gain: 0; mask: 0xffffffff)'
```

Comment:
This means: ``nan`` is the default for proc data (float32), while 0 is the default for raw data (uint16), correct? There is no nan-equivalent for integers, I guess, but still 0 has quite a different meaning as Thomas pointed out. I know that 0 is exactly what we want as work-around to make CrystFEL happy, but still ...

Reply:
yep, there is no `nan` for integers. I figured `0` would be the most appropriate equivalent there?

Reply:
Practically, yes, for many scenarios. For crystallography (CrystFEL in particular) I'd tend to prefer a (absolutely spoken: large) negative value - but a reasonable default would be very situation-dependent and difficult to set - hence, the specific user value becomes important. And zero is still the best value for a default.

Reply:
There are no negative values for `uint16` (the `u` means unsigned, i.e. no way to represent negative numbers). One convention is to use the largest possible number, 65535. But I think 0 is the more obvious default.

I think the help string should say that the distinction is between corrected & raw data, though; it's not necessarily obvious why there are two defaults for data or which way round they go.

Reply:
Yes, you are correct with uint16 - actually I was referring to proc treatment with SFX application in mind (because that's what I am testing now: CrystFEL with VDS where -10000 is used as fillvalue). So for proc with float32 it's possible, but I'd leave the default at nan. And let the default for raw be 0, as I wrote.

Reply:
Alright. I've updated the docstring with proc/raw for data.
We seem to all agree on the defaults \o/